tokens/0x3a371d4fff12931ff784c18df7ea6741b0fa971f.yaml

```diff
@@ -0,0 +1,14 @@
+---
+addr: '0x3a371d4fff12931ff784c18df7ea6741b0fa971f'
+decimals: 18
+description: >-
+  Veritaseum enables software-driven P2P capital markets without brokerages, banks or traditional exchanges
```

Comment:
@therealMikkoK Description for this token should probably include a bit more on this specific token's use.